vocabularies/Org.OData.JSON.V1.xml

```diff
@@ -14,64 +14,202 @@
+            <PropertyValue Property="href"
+              String="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.JSON.V1.json" />
           </Record>
           <Record>
             <PropertyValue Property="rel" String="describedby" />
-            <PropertyValue Property="href" String="https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.JSON.V1.md" />
+            <PropertyValue Property="href"
+              String="https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.JSON.V1.md" />
           </Record>
         </Collection>
       </Annotation>
 
-      <Term Name="Schema" Type="JSON.JSON" Nullable="false" AppliesTo="EntityType Parameter Property ReturnType Term TypeDefinition">
+      <Term Name="Schema" Type="JSON.JSON" Nullable="false"
+        AppliesTo="EntityType Parameter Property ReturnType Term TypeDefinition">
         <Annotation Term="Core.RequiresType" String="Edm.Stream" />
-        <Annotation Term="Core.Description" String="The JSON Schema for JSON values of the annotated media entity type, property, parameter, return type, term, or type definition" />
+        <Annotation Term="Core.Description"
+          String="The JSON Schema for JSON values of the annotated media entity type, property, parameter, return type, term, or type definition" />
         <Annotation Term="Core.LongDescription">
-          <String>The schema can be a schema reference, i.e. `{"$ref":"url/of/schemafile#/path/to/schema/within/schemafile"}`</String>
+          <String>The schema can be a reference, i.e. `{"$ref":"url/of/schemafile#/path/to/schema/within/schemafile"}`</String>
         </Annotation>
       </Term>
 
+      <Function Name="query" IsBound="true" IsComposable="true">
+        <Annotation Term="Core.Description"
+          String="Query a stream value of media type `application/json`, returning a stream value of media type `application/json`" />
+        <Annotation Term="Core.LongDescription">
+          <String>Extracts a JSON value, such as an array, object, or a JSON scalar value (string, number, boolean, or `null`) from the `input` JSON value:
+- If `path` only consists of the root identifier followed by name and index selectors, it returns the identified single node within `input`, or `null` if no node is identified. 
+- If `path` potentially identifies multiple nodes within `input` (by using descendant, wildcard, union, array subset, or filter selectors), it returns an array containing the identified nodes, or an empty array if no node is identified. 
+- If `input` is not a valid JSON value, the function returns `null`.
+- If `path` is `null`, not a valid [JSONPath expression](#Path), or does not match the structure of `input` (for example applying an index selector to a scalar value), the function returns `null`. 
+          </String>
```

Comment:
```suggestion

            The SQL function `JSON_QUERY` returns the stringified version of the extracted JSON value.
          </String>
```

Reply:
I'd rather add this to a new section/document on "implementation options", not to the definition of the OData functions, which may be implemented in a number of different ways on different platforms.